guides/common/modules/proc_enabling-pull-based-transport.adoc

```diff
@@ -0,0 +1,66 @@
+[id="enabling-pull-based-tranport_{context}"]
+= Enabling Pull-based Transport
+
+If your {Project} or {SmartProxy} is preinstalled, use the following procedure to enable pull-based transport.
+
+.Procedure
+
+. To install the remote execution script, run:
+ifdef::satellite[]
+----
+{satellite-installer} --foreman-proxy-plugin-remote-execution-script-mode=pull-mqtt
+----
+endif::[]
+ifdef::foreman,orcharhino,katello[]
+----
+{foreman-installer} --foreman-proxy-plugin-remote-execution-script-mode=pull-mqtt
+----
+endif::[]
+
+// Start this list from 2.
+[start=2]
+. To enable the {SmartProxy} for host registration, run:
+
+----
+{foreman-installer} --foreman-proxy-templates=true --foreman-proxy-registration=true
+----
+
+// Start this list from 3.
+[start=3]
+. To enable the pull provider, run:
```

Comment:
```suggestion
. To enable the pull mode, run:
```

Reply:
+1; there are multiple instances of that phrase around